src/app/services/support.service.ts

```diff
@@ -28,4 +28,11 @@ export class SupportService {
       map(response => response),
       catchError((error: Response) => observableThrowError(error)));
   }
+
+  public async downloadSupportBundle(bundle): Promise<Blob> {
+    const file = await this.http.get<Blob>(
+      this.SUPPORT_BUNDLE_URL + '/' + bundle,
+      { responseType: 'blob' as 'json' }).toPromise();
```

Comment:
as json? please add docstring

Reply:
is http interceptor sending the token here?

Reply:
Type script throws an error `Types of property 'responseType' are incompatible.
Type '"blob"' is not assignable to type '"json"` if `blob as json` not used.


Yes all request have token supplied by  http interceptor